policy_wizard/utils.py

```diff
@@ -65,3 +65,10 @@ def validate_request(request):
     lti_object = LTI('initial', 'any')
 
     return lti_object._verify_request(request)
+
+# Inactivates active policies for a particular course
+def inactivate_active_policies(request):
+    policies_to_inactivate = Policies.objects.filter(course_id=request.session['course_id'], is_active=True)
+    policies_to_inactivate.update(is_active=False)
+    for item in policies_to_inactivate:
+        item.save()
```

Comment:
Are you looping through and calling `save` here in order to update the `updated_at` attribute? Otherwise, the `update` method should be enough to bulk update the queryset.

Reply:
Agree with @dodget here. Per [the docs](https://docs.djangoproject.com/en/3.1/ref/models/querysets/#update) and [source code](https://github.com/django/django/blob/master/django/db/models/query.py#L765), this should apply the change to all of the objects in the queryset (issues a SQL UPDATE). I think the only reason to call `save()` is if there's some logic that needs to be triggered on save for each item.

Reply:
Agreed. I have removed the unnecessary save operation. Thanks.